opencti-platform/opencti-graphql/src/config/conf.js

```diff
@@ -276,6 +287,20 @@ export const logTelemetry = {
   }
 };
 
+export function shutdownLoggers() {
+  const shutdownPromises = [appLogger, auditLogger, supportLogger].map(
+    (logger) => new Promise(
+      (resolve) => {
+        logger
+          .end()
+          .on('finish', resolve);
+      }
+    )
+  );
+
+  return Promise.all(shutdownPromises);
+}
```

Comment:
Could you also condition this shutdown method with a configuration variable? Maybe something like `enable_logs_safe_shutdown` in the app config